docs/api.adoc

```diff
@@ -7,58 +7,188 @@ v2.0.0, 2016-12
 
 Detailed description of api docs here!
 
-Hosts
+Search
 -----
-
 [nested]
-* List all
+* Search for host or group.
+Specify type as  a group, host, or any to limit the search.
 +
 [source]
 ....
-  /api/host/list
+GET /api/search?type=any&name=web_hosts
 ....
 ** Example results:
 +
 [source,json]
 ----
 {
+  "status": "success",
+  "code": 200,
```

Comment:
http status codes should not be in the JSON too, since they'll already be in the HTTP header